ui/app/common/obs/models/observation.js

```diff
@@ -82,8 +92,12 @@ Bahmni.Common.Obs.Observation = (function () {
                 return this.complexData.display;
             }
 
-            if (this.groupMembers.length > 1 && this.formNamespace != null) {
-                return this.groupMembers[0].value.name + " " + "since" + " " + this.groupMembers[1].value + " " + this.groupMembers[2].value.name;
+            if (this.isConceptNameChiefComplaintData()) {
+                if (this.groupMembers[0].value.name !== this.translate.instant("CHIEF_COMPLAINT_DATA_OTHER_CONCEPT_KEY")) {
+                    return this.groupMembers[0].value.name + " " + this.translate.instant("SINCE_KEY") + " " + this.groupMembers[1].value + " " + this.groupMembers[2].value.name;
+                } else {
+                    return this.groupMembers[0].value.name + ' (' + this.groupMembers[1].value + ") " + this.translate.instant("SINCE_KEY") + " " + this.groupMembers[2].value + " " + this.groupMembers[3].value.name;
```

Comment:
Translation strings can't be created using concatentation, becase grammar is different in different languages, and concatentation breaks that.

Please use a single translation sentence with placeholders.